pages/docs/[slug].js

```diff
@@ -47,9 +48,16 @@ function DocPage(props) {
 export default DocPage;
 
 export const getStaticProps = async ({ params }) => {
-  const sideNavFiles = await staticRequest({
+  const tinaProps = await getStaticPropsForTina({
```

Comment:
WOuld something like [this](https://github.com/tinacms/tinacms.org/pull/1063/files#diff-aa42e9fab5e9ed03800898ab696b89cdfd1762c33932a1b9916a30b700125d4fR34) work here?

For context: https://github.com/tinacms/tinacms/issues/2108

Reply:
Not 100% sure what you mean? If you mean do something like

```ts
const sideNavFiles = await staticRequest(...)
const sideNav = ProcessParseSIdeNav(sideNavFiles)
return {
 props{
 }
}
```

Reply:
Will swallowing the errors in a catch block work for now? https://github.com/tinacms/tinacms.org/pull/1063/files#diff-aa42e9fab5e9ed03800898ab696b89cdfd1762c33932a1b9916a30b700125d4fR41

Reply:
And yeah I just mean that I believe getStaticPropsForTIna was intentionally removed from the starter https://github.com/tinacms/tinacms/issues/2108

Reply:
@jamespohalloran Makes sense to me, I pushed up those changes.